shared/src/helpers/get-badge-icon/get-badge-icon.ts

```diff
@@ -3,9 +3,12 @@ import { BsaBadgesTypeEnum } from '../../index.js';
 
 const getBadgeIcon = (type: ValueOf<typeof BsaBadgesTypeEnum>): string => {
     const values = {
-        [BsaBadgesTypeEnum.BSA_SCORE]: 'headphones',
-        [BsaBadgesTypeEnum.PERSONALITY]: 'account-multiple-outline',
-        [BsaBadgesTypeEnum.SOFT_SKILLS]: 'account-star-outline',
+        [BsaBadgesTypeEnum.ENGLISH_LEVEL]: 'headphones',
+        [BsaBadgesTypeEnum.PROJECT_SCORE]: 'headphones',
+        [BsaBadgesTypeEnum.LECTURE_SCORE]: 'headphones',
+        [BsaBadgesTypeEnum.BEST_LECTURE_SCORE]: 'headphones',
+        [BsaBadgesTypeEnum.TEAM_SCORE]: 'headphones',
+        [BsaBadgesTypeEnum.COMMUNICATION_SCORE]: 'headphones',
```

Comment:
Why we have the same icons for badges ? The same as design ?

Reply:
Yes, they are the same in the figma